components/06-examples/beheer-basis.config.yml

```diff
@@ -0,0 +1,60 @@
+notes: |
+status: wip
+context:
+  menu: 
+    modifier: main
+    items: 
+      - label: Menu titel A
+        active: true
+      - label: Menu titel B
+      - label: Menu titel C
+  breadcrumbs:
+    crumbs:
+      - label: Home
+        url: '#'
+      - label: Menu titel A
+        url: '#'
+  title: Overzicht samenwerkingen
+  sections:
```

Comment:
Dit lijkt op een formulier specificatie, maar ik zie geen formulier? Overblijfsel van een copy/paste ?

Reply:
Is inderdaad overblijfsel van copy/paste. Ga ik opschonen.